timeline/templates/timeline/_sort_header.html

```diff
@@ -0,0 +1,21 @@
+<div class="card sodar-sort-card">
+  <div class="card-header">
+    <h4>
+      <i class="iconify"
+         data-icon="mdi:sort-clock-ascending"></i>
+      Sort options
```

Comment:
"Timeline Events" would be an appropriate title for the card. Also note that [title case](https://research.arizona.edu/faq/what-do-you-mean-when-you-say-use-title-case-proposalproject-titles) should be used for all titles. I probably haven't specified this in contibuting docs, will do so when I do cleanup for the next release :)